.github/workflows/r_valgrind.yml

```diff
@@ -0,0 +1,30 @@
+name: R valgrind tests
```

Comment:
/gha run r-valgrind-tests

Reply:
^ this comment triggered this: https://github.com/microsoft/LightGBM/runs/1262655120?check_suite_focus=true

Reply:
this succeeded! https://github.com/microsoft/LightGBM/runs/1262655120?check_suite_focus=true

Reply:
/gha run r-valgrind

Reply:
/gha run-r-valgrind

Reply:
^ @StrikerRUS notice that this comment I just left triggered a run that was skipped, because it wasn't part of a review:

https://github.com/microsoft/LightGBM/actions/runs/312992159

Reply:
/gha run r-valgrind

Reply:
^ but this comment triggered a real run because it was part of a  review

https://github.com/microsoft/LightGBM/actions/runs/312993170

I see now that I am able to keep responding to a thread with review comments as long as it's on the "Files Changed" tab (like you mentioned), so I think it's ok!

Reply:
@jameslamb 
Please notice that
`/gha run-r-valgrind` != `/gha run r-valgrind`